db/schema.rb

```diff
@@ -34,7 +48,7 @@
     t.index ["pipeline_output_id"], name: "index_backgrounds_pipeline_outputs_on_pipeline_output_id"
   end
 
-  create_table "backgrounds_pipeline_runs", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=utf8 COLLATE=utf8_unicode_ci" do |t|
+  create_table "backgrounds_pipeline_runs", force: :cascade, options: "ENGINE=InnoDB DEFAULT CHARSET=latin1" do |t|
```

Comment:
Per @tfrcarvalho:
You can prevent these changes by changing your local db as follows:
```
mysql -h db -u root --database= idseq_development -B -N -e "SHOW TABLES" | awk '{print "ALTER TABLE", $1, "CONVERT TO CHARACTER SET utf8 COLLATE utf8_unicode_ci;"}' | mysql -h db -u root --database= idseq_development;
```

In rails dbconsole:
```
ALTER TABLE users MODIFY email varchar(255) COLLATE utf8_unicode_ci;
ALTER TABLE users MODIFY name varchar(255) COLLATE utf8_unicode_ci;
ALTER TABLE users MODIFY encrypted_password varchar(255) COLLATE utf8_unicode_ci;
ALTER TABLE users MODIFY reset_password_token varchar(255) COLLATE utf8_unicode_ci;
ALTER TABLE users MODIFY current_sign_in_ip varchar(255) COLLATE utf8_unicode_ci;
ALTER TABLE users MODIFY last_sign_in_ip varchar(255) COLLATE utf8_unicode_ci;
ALTER TABLE users MODIFY authentication_token varchar(30) COLLATE utf8_unicode_ci;
```